src/sdk/namespace_blob.js

```diff
@@ -422,15 +432,24 @@ class NamespaceBlob {
                 // clear count for next updates
                 count = 0;
             });
-            res = await P.fromCallback(callback =>
-                this.blob.createBlockFromStream(
-                    block_id,
-                    this.container,
-                    params.key,
-                    params.source_stream.pipe(count_stream),
-                    params.size, // streamLength really required ???
-                    callback)
-            );
+            try {
+                res = await P.fromCallback(callback =>
+                    this.blob.createBlockFromStream(
+                        block_id,
+                        this.container,
+                        params.key,
+                        params.source_stream.pipe(count_stream),
+                        params.size, // streamLength really required ???
+                        callback)
+                );
+            } catch (err) {
+                object_sdk.rpc_client.pool.update_issues_report({
+                    namespace_resource_id: this.namespace_resource_id,
+                    error_code: err.rpc_code || err.code,
```

Comment:
Take the error code instead of rpc_code, because you do not check rpc_code errors in `update_issues_report`

Reply:
correct. deleted